### DISCUSSION TITLE
How to query the result of receipt from the database?

### DISCUSSION THREAD

**Discussion by WuTao18** [near/near-indexer-for-explorer#199] [Q&A] [answered]
For example, https://explorer.near.org/transactions/38CyUF2yiLq4DSx23kS12g4M3Z19dwDQ7FVAmEJvxdER has two results: "0" and "500000000000000000000000", how to query these two items from the database?
![image](https://user-images.githubusercontent.com/32795326/143895125-2219aeba-d29a-4915-9e97-6366c86bf5b6.png)
![image](https://user-images.githubusercontent.com/32795326/143895169-c281258b-abc6-438e-a843-ea27f3453ec7.png)

**Comment by frol** [comment, marked as answer]
@WuTao18 We don't store neither results nor logs in the database since that is not something we need to query on (filter or order by, etc). We only fetch those values on the Transaction Details page, and we use archival JSON RPC for that.
